Person.rb

```diff
@@ -27,6 +28,11 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
```

Comment:
Good effort, @Estete9! 👍 Your implementation of the `add_rental` method demonstrates an understanding of the many-to-many relationship. However, there's a small adjustment needed to fully align with the project requirements.

#### Suggested Improvement:

   - The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association.

#### Updated Method:

```ruby
  # *A method for adding rentals should be present. Link to the example in a previous lesson: https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship
  def add_rental(book, date)
    # *The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association
    Rental.new(date, book, self)
  end
```

#### Explanation:
   - This adjustment ensures that the `add_rental` method conforms to the specified project requirements.
   - By including the `person` parameter in the method, you establish the 'has-many' association with the correct parameters.

Keep up the good work! If you have any questions or need further clarification, feel free to ask. 🚀

Best,
Temitope

Reply:
@topeogunleye My friend again, is this for the person.rb file? both comments you made are for the same function in the same file, I would greatly appreciate it if you can give me further insight on this. Thank you